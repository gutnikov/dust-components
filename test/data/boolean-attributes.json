{
  "state1" : true,
  "state2" : false
}