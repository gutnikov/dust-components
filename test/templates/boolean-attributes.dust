<div>
	{@Button className="simple-button" disabled=state1 text="I'm a button"/}
	{@Button className="simple-button" disabled=state2 text="I'm a button"/}
</div>
