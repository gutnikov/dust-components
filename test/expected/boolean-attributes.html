<div>
    <button class="button simple-button" disabled>I'm a button</button>
    <button class="button simple-button">I'm a button</button>
</div>
